packages/wonder-blocks-dropdown/utils/types.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+// TODO(sophie): incorporate this stuff later
+// export type ActionItemProps = {
+//     type: "action",
+//     /** Whether this item is disabled. Default false. */
+//     disabled?: boolean,
+//     /** Display text of the item. */
+//     label: string,
+//     /** URL to navigate to. */
+//     href?: string,
+//     /** Whether to use client-side navigation. */
+//     clientNav?: boolean,
+//     /** Callback on the action. */
+//     onClick?: () => void,
+// };
+
+export type SelectItemProps = {
+    type: "select",
+    /** Whether this item is disabled. Default false. */
+    disabled?: boolean,
+    /** Display text of the item. */
+    label: string,
+    /** Value of this item. Treat as a key. */
+    value: string,
+    /** Optional extra callback. Passes whether this item is selected. */
+    onClick?: (selected: boolean) => void,
+};
+
+// export type SeparatorProps = {
+//     type: "separator",
+// };
```

Comment:
Should this be used somewhere since we do have separators in some of the examples?

Reply:
This will be used in the upcoming `ActionMenu` pull request. I just didn't want to have to save these somewhere else when I committed the `types.js` file.